src/inputs/labeled-inputs.scss

```diff
@@ -169,6 +167,45 @@
   }
 }
 
+/// Wrapping
+@mixin iui-input-label-wrapper {
+  @include iui-input-label-styling;
+  grid-area: label;
+  align-self: center;
+}
+
+/// Non-wrapping
```

Comment:
Would be nice to document what this mixin is for, what the modifiers are, and what's the difference between default and inline.

Reply:
I updated the descriptions a little bit but probably not descriptive enough.